service/src/main/java/io/camunda/service/JobServices.java

```diff
@@ -12,13 +12,20 @@
 import io.camunda.service.transformers.ServiceTransformers;
 import io.camunda.zeebe.broker.client.api.BrokerClient;
 import io.camunda.zeebe.gateway.impl.broker.request.BrokerActivateJobsRequest;
+import io.camunda.zeebe.gateway.impl.broker.request.BrokerFailJobRequest;
 import io.camunda.zeebe.gateway.impl.job.ActivateJobsHandler;
 import io.camunda.zeebe.gateway.impl.job.ResponseObserver;
+import io.camunda.zeebe.protocol.impl.record.value.job.JobRecord;
 import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
 import java.util.function.Consumer;
 
 public final class JobServices<T> extends ApiServices<JobServices<T>> {
 
+  public static final Integer DEFAULT_RETRIES_VALUE = 0;
```

Comment:
I'm not sure about this approach, I think it's possible to create those static variables in the generated class. But before doing that I want to know your opinion on this approach @tmetzke

Reply:
I'm not sure if we need those default values at all, actually.
1. The generated REST model class `JobFailRequest` already contains initialized attributes with the default values defined in the YAML.
2. In the `RequestMapper`, we could by default map `null` numeric values to 0, just like gRPC does. It will be hard to find a case where it should be something other than 0 by default.

Reply:
For number maybe you are right, but what about other types ?

Reply:
Let's talk about that when we get there, I guess 🙂